### DISCUSSION TITLE
Filament Sensor

### DISCUSSION THREAD

**Discussion by stefanelul2000** [Jyers/Marlin#300] [Q&A] [answered]
Hello,

I would like to know if filament sensor is enable in the FW. In this case I am interested in the smart filament sensor. In my case the sensor is connected to the Raspberry Pi (Octoprint) not the printer directly. 

Thanks

**Comment by Jyers** [comment, marked as answer]
The filament sensor option is enabled by default, so the filament sensor itself can be enabled from the advanced menu. However, if you are using a sensor connected to your RPi I don't think any firmware adjustments are needed, octoprint will do all the work.
